index.js

```diff
@@ -14,12 +14,22 @@ module.exports = app => {
       'no-changes-template': `* No changes`,
       'version-template': `$MAJOR.$MINOR.$PATCH`,
       categories: [],
-      'exclude-labels': []
+      'exclude-labels': [],
+      replacers: []
     }
     const config = Object.assign(
       defaults,
       (await getConfig(context, configName)) || {}
     )
+    config.replacers = config.replacers.filter(replacer => {

```

Comment:
Would it be an idea to extract this to a separate function called `validateReplacers` as to keep the main function clean and it would be self-documenting instead of having to read the code? :)